package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "spotify-web-helper",
-  "version": "1.14.0",
+  "version": "2.0.0",
```

Comment:
The changes here are significant; however, unless they actually force some important new requirement or other end user changes, updating the major version number could be avoided to allow users to continue to use this repo with the new features without changing their package files.